### DISCUSSION TITLE
lightttpd error:

### DISCUSSION THREAD

**Discussion by ssagerian** [AllskyTeam/allsky-modules#58] [General]
2023-12-30 08:58:34: mod_fastcgi.c.487) FastCGI-stderr:PHP 

Warning:  copy(/home/pi/allsky/config/modules/postprocessing_day.json-last): failed to open stream: Permission denied in /home/pi/allsky/html/includes/moduleutil.php on line 347

all the module files have pi : pi as user group, is that the issue?

pi@homesky:~/allsky/config_repo/modules $ ls -l
total 24
-rw-r--r-- 1 pi pi  370 Nov 24 15:20 module-settings.json
-rw-r--r-- 1 pi pi 2280 Nov 24 15:20 postprocessing_day.json
-rw-r--r-- 1 pi pi    3 Nov 24 15:20 postprocessing_daynight.json
-rw-r--r-- 1 pi pi    3 Nov 24 15:20 postprocessing_nightday.json
-rw-r--r-- 1 pi pi 2284 Nov 24 15:20 postprocessing_night.json
-rw-r--r-- 1 pi pi    3 Nov 24 15:20 postprocessing_periodic.json
p

**Comment by Alex-developer** [comment]
They should be 

`pi:www-data`

Try changing the permissions and see it it helps
